src/main/java/core/basesyntax/enums/Figures.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.enums;
+
+public enum Figures {
+    CIRCLE,
+    ISOSCELES_Trapezoid,
```

Comment:
uppercase for that pls
```suggestion
    ISOSCELES_Trapezoid,
```